### ISSUE TITLE
Tensorflow Custom Object Detection Training Time

### ISSUE THREAD

**Issue by tornadots** [tensorflow/models#4867]
Hi all 

I tried to train a frcnn for detecting money it has 3 classes.

I used this link
https://github.com/EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10
to generate something.
And it uses faster_rcnn_inception_v2_coco_2018_01_28 config

It takes too much time how can i fix this or is this normal ?

INFO:tensorflow:global_step/sec: 0.049895
INFO:tensorflow:Recording summary at step 1830.
INFO:tensorflow:Recording summary at step 1830.
INFO:tensorflow:global step 1831: loss = 0.2820 (22.984 sec/step)
INFO:tensorflow:global step 1831: loss = 0.2820 (22.984 sec/step)
INFO:tensorflow:global step 1832: loss = 0.2418 (20.894 sec/step)
INFO:tensorflow:global step 1832: loss = 0.2418 (20.894 sec/step)
INFO:tensorflow:global step 1833: loss = 0.1033 (19.598 sec/step)
INFO:tensorflow:global step 1833: loss = 0.1033 (19.598 sec/step)
INFO:tensorflow:global step 1834: loss = 0.4120 (21.157 sec/step)
INFO:tensorflow:global step 1834: loss = 0.4120 (21.157 sec/step)

my tensorflow version b'v1.9.0-0-g25c197e023' 1.9.0
Cuda compilation tools, release 9.1, V9.1.85

My computer has 8gb ram GTX 950M and 4gb Memory 
2.6Ghz i7-6700HQ

**Comment by tornadots** [comment]
Yes  you're right. Before I totally uninstalled  tensorflow I tried to upgrade it to gpu version but it didn't work  but now it is working and this situation is brutal :) Now it takes just 0.5 sec for each step =)

**Comment by karmel** [comment]
Thanks for helping, @HyperWayne -- closing this issue as it sounds like it has been resolved.

**Comment by MFR1** [comment]
Hi There, I am getting 0.7 seconds while detection with GPU but 2.2 seconds with CPU, can anyone help to get better speed?
